about/people.yml

```diff
@@ -366,3 +366,11 @@ data:
     brown_directory_uuid: 77e8343c-498a-4bed-bedf-ed0a59d0fa3c
     bio: Galen is passionate about building usable, flexible, and accessible web applications. Outside of work, he loves playing music, playing tennis and squash, and hiking. 
     image: galen_main.jpg
+  - name: Rain Fan
+    type: Full Time
+    team: CCV Operations
+    title: Research Computing Consultant
+    github_username: rainxfan
+    brown_directory_uuid: D5bc11b8-b031-4402-8f40-d08ccee18b60
+    bio:
+    image: rain_main.jpg
```

Comment:
It looks like the file extension here should be `.jpeg`. Either that, or we should rename the image itself.